language-server/modules/langserver-core/src/test/resources/completion/module_part_context/config/module_level_after_configurable_config3.json

```diff
@@ -678,7 +678,7 @@
       "detail": "Snippet",
       "sortText": "D",
       "filterText": "public_function_main",
-      "insertText": "public function main() {\n\t${1}\n}",
+      "insertText": "function main() {\n\t${1}\n}",
```

Comment:
We should check this case too. `configurable <cursor>`, here we aren't adding the `public` keyword. I don't think we should suggest the main function altogether here. WDYT?

Reply:
+1